perf-measures/bundle-check/scripts/process-results.ts

```diff
@@ -1,14 +1,23 @@
-import * as fs from 'node:fs/promises'
+import * as fs from 'node:fs'
 
 async function main() {
-    const currentResult = (await fs.readFile('./generated/after.js')).byteLength
-    let previousResult: number | null = null
-    try {
-        previousResult = (await fs.readFile('./generated/before.js')).byteLength
-    } catch (e) {}
-    const table = ['| | Current | Previous |', '| --- | --- | --- |']
-    table.push(`| Bundle Size | ${currentResult} | ${previousResult || 'N/A'} |`)
-    console.log(table.join('\n'))
+    const args = process.argv.slice(2);
+    const filePath = args[0];
```

Comment:
Allowing file paths to be specified seems to be more versatile.
Or, it would be easier to understand the connection to build in this source.

```ts
import { build, Format } from 'esbuild';

const buildOptions = {
  entryPoints: ['dist/index.js'],
  bundle: true,
  minify: true,
  format: 'esm' as Format,
  outfile: 'perf-measures/bundle-check/generated/bundle.js'
};

build(buildOptions).then(() => {
  console.log('Build succeeded.');
  // process result
}).catch(() => {
  console.error('Build failed.');
});

```

Reply:
I don't understand the build pipeline, are the build options specified the same as those in the build pipeline? Just curious.

Reply:
I don't know if there is a need, but we could try to find out the size for each of the major build options.

Reply:
@k2tzumi 

For the `hono` package, distributed files are not minified. We assume the bundle size when a user minifies an application using `hono`. For example, `wrangler deploy --minify`. So, the options depend on the user. Using esbuild and the options you specified are generic, so I think that's fine.

Reply:
> Or, it would be easier to understand the connection to build in this source.v

Does this mean we can remove this line if you add the build command in `perf-measures/bundle-check/scripts/process-results.ts`? If so, it's good.

https://github.com/honojs/hono/pull/3491/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR226

Reply:
> For the hono package, distributed files are not minified.

Would you also like to know the size of the bundled sources that are not minified?
Is it sufficient to check the size of the minified source only as it is now?

> Does this mean we can remove this line if you add the build command in perf-measures/bundle-check/scripts/process-results.ts? If so, it's good.

Yes.
We can port a BUILD in a separate command to a script.
We believe that it is better to do the size comparison with multiple BuildOptions, but on the script side.

Reply:
> Would you also like to know the size of the bundled sources that are not minified?

No. We only have to know minified file sizes because we focus on them when they are minified.

> Yes.
> We can port a BUILD in a separate command to a script.
> We believe that it is better to do the size comparison with multiple BuildOptions, but on the script side.

I see! It makes sense!